app/assets/stylesheets/custom.css.scss

```diff
@@ -133,35 +139,40 @@ body {
   padding: 0;
   font-size: 4.5em;
   font-weight: 700;
-  color: gray;
+  color: $muted;
 }
 
 /* Custom Button Styles */
 
-.btn-dark {
-  border-radius: 0;
-  color: $white;
-  background-color: rgba(0,0,0,0.4);
+.btn-xlarge {
+  padding: 18px 28px;
```

Comment:
Properties should be ordered -moz-border-radius, -webkit-border-radius, border-radius, font-size, line-height, padding